AutomatedTesting/Registry/asset_cache_server_settings.setreg

```diff
@@ -0,0 +1,32 @@
+{
+    "O3DE": {
+        "AssetProcessor": {
+            "Settings": {
+                "Server": {
+                    "cacheServerAddress": "",
```

Comment:
Inconsistent casing here. We should should follow consistent casing ie Name, Glob etc Does setreg have a standard?

Reply:
I guess the keys could be Capital, but I am not aware of any casing format standard